protocols/bgp/server/fsm.go

```diff
@@ -95,7 +95,7 @@ func NewPassiveFSM(peer *peer, con *net.TCPConn) *FSM {
 	return fsm
 }
 
-// NewActiveFSM initiates a new passive FSM
```

Comment:
It looks like it was a typo but I think it was not. The BGP active state means the things is actively waiting for incoming connections and thus is passive.

From RFC4271:
 Active State:

      In this state, BGP FSM is trying to acquire a peer by listening
      for, and accepting, a TCP connection.